bin/logstash.lib.sh

```diff
@@ -140,7 +132,9 @@ setup_ruby() {
 jruby_opts() {
   printf "%s" "--1.9"
   for i in $JAVA_OPTS ; do
-    printf "%s" " -J$i"
+    if [ "x$i" != "x" ] && [ "x$i" != "x " ]; then
```

Comment:
I got some weird errors when testing shell options, but that may have been from something else.  I'll retest with this as just `-z`